kvm-install-vm

```diff
@@ -434,7 +434,11 @@ function create_vm ()
 
     # cloud-init config: set hostname, remove cloud-init package,
     # and add ssh-key
-    cat > $USER_DATA << _EOF_
+    if [ ! -z "${USERDATA+x}" ]
```

Comment:
Why not just use `-n` instead of `! -z`?  Also, why the `+x` at the end?

Reply:
Yeah, `-n` will do and `+x` is not needed either.